azavea/raster-foundry
# Feature/lk/new project ui panels

## Overview

Add routes for the new project workflow
New color mode selection view
New integrated browse / project scene views
Integrated color correction

All of the features act on the same map (except scene preview) and share some common state

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo

![image](https://cloud.githubusercontent.com/assets/4392704/24977052/d3022888-1f99-11e7-95e2-660ee97c5922.png)
![image](https://cloud.githubusercontent.com/assets/4392704/24977065/dd033ade-1f99-11e7-9bf7-3abc8a564282.png)
![image](https://cloud.githubusercontent.com/assets/4392704/24977076/e545b500-1f99-11e7-835f-94e13f6d6f6c.png)
![image](https://cloud.githubusercontent.com/assets/4392704/24977085/ebddf21a-1f99-11e7-8781-b3f47fcce4f7.png)
![image](https://cloud.githubusercontent.com/assets/4392704/24977092/f0b0e5a4-1f99-11e7-81a8-9e36cc411a2b.png)
![image](https://cloud.githubusercontent.com/assets/4392704/24977095/f60cfb0a-1f99-11e7-8dcc-177f8a6b232e.png)
![image](https://cloud.githubusercontent.com/assets/4392704/24977107/fc3d1a96-1f99-11e7-896f-6abed15da923.png)


### Notes

There is a known bug with the color correct which causes disabled color correct sliders to not have the disabled property set correctly - it doesn't affect their function though (any changes are reset immediately)

I've left some of the old templates in place until we actually implement the corresponding project views (such as scene re-ordering, masking, etc)

## Testing Instructions

* Verify that the following work as expected and all work on the same map:
  * Viewing / deleting project scenes
  * Adding scenes to a project
  * Previewing scenes in the browse view
  * Deleting projects
  * Selecting a color mode
  * Color correcting scenes, individually and as a group
* Keep an eye out for any map state that carries between modes (such as footprints, etc) that shouldn't

Closes #1440 
Closes #1437 
Closes #1438
